<?xml version="1.0" encoding="UTF-8"?>
<experiment>
    <failure-description>
        <failure-description test-name="Local search" description="The Execution crushed with the exception: bgu.dcr.az.mas.exp.ExperimentExecutionException: No problem generator is defined at bgu.dcr.az.mas.cp.CPExperimentTest.execute(CPExperimentTest.java:179) at bgu.dcr.az.mas.cp.CPExperimentTest.execute(CPExperimentTest.java:155) at bgu.dcr.az.mas.cp.CPExperiment.execute(CPExperiment.java:102) at bgu.dcr.az.ui.AppController.main(AppController.java:77) " execution-number="0" algorithm="CONC_FB2"/>
    </failure-description>
    <tests>
        <test seed="42" name="Local search" env="async">
            <algorithms>
                <algorithm name="CONC_FB2"/>
                <algorithm name="CONC_FB"/>
            </algorithms>
            <loop>
                <for-loop repeat-count="100" run-var="p1" start="0.1" end="0.9" tick-size="0.1">
                    <looper>
                        <do-once/>
                    </looper>
                </for-loop>
            </loop>
            <statistics>
                <msgc-sc type="BY_RUNVAR"/>
                <item>Number of cores in use</item>
                <item>Algorithm CPU time spending</item>
                <cc-sc/>
            </statistics>
        </test>
    </tests>
    <Statistics/>
</experiment>
